cmd/crowdsec-cli/api.go

```diff
@@ -189,7 +185,6 @@ cscli api credentials   # Display your API credentials
 			}
 			fmt.Printf("machine_id: %s\n", outputCTX.API.Creds.User)
 			fmt.Printf("password: %s\n", outputCTX.API.Creds.Password)
-			return
```

Comment:
For my own curiosity, I wonder why why this pattern is not linter compliant. I thought put return was a best practice.

Reply:
code simplification : return doesn't add anything